README.md

```diff
@@ -93,5 +94,111 @@ When you have any questions, find a bug or have an idea for a feature you can cr
+| Sie sind nicht sehr gut für die Umwelt, da sie nach nur einem Schuljahr weggeworfen werden               | Dlool ist digital, es wird kein Papier benötigt.                                                                                                                                                                                                                                                       |
+| Sie sind nicht sehr praktisch                                                                            | Dlool ist auf allen Geräten verfügbar.                                                                                                                                                                                                                                                                 |
+| Einträge sind nicht sehr strukturiert                                                                    | Dlool hat ein strukturiertes System zum Eintragen von Hausaufgaben.                                                                                                                                                                                                                                    |
+| Wenn Sie Ihr Hausaufgabenheft verlieren, sind alle Ihre Daten für immer verloren                         | Dlool ist digital, Sie können es nicht verlieren.                                                                                                                                                                                                                                                      |
+| Jedes Jahr braucht man ein neues Hausaufgabenheft                                                        | Dlool kann für mehrere Jahre verwendet werden.                                                                                                                                                                                                                                                         |
+| Sie sind an eine bestimmte Marke gebunden                                                                | Dlool ist kostenlos und quelloffen. *(Mit Dlool können Sie Ihre Daten in ical oder todo.txt exportieren. Außerdem gibt es [eine Möglichkeit](https://github.com/Dlurak/dlool-scriptable/blob/main/reminders/README.md), Dlool mit Apple Reminders zu synchronisieren.)* **(noch nicht implementiert)** |
+| Hausaufgaben, Tests und Notizen sind nicht getrennt                                                      | Dlool hat ein anderes System für Hausaufgaben, Tests und Notizen. Alles ist getrennt organisiert, aber dennoch leicht an einem Ort zu finden.                                                                                                                                                          |
+
+---
+
+## Funktionen
```

Comment:
I'm not sure if *Features* would be a better fit. But we can also leave it as *Funktionen* if you think so.